platform/linux_platform_test.go

```diff
@@ -2700,6 +2700,38 @@ Number  Start   End     Size    File system  Name             Flags
 		})
 	})
 
+	Describe("MigratePersistentDiskWithReadaheadOverride", func() {
+		var mounter *fakedisk.FakeMounter
+		BeforeEach(func() {
+			mounter = diskManager.FakeMounter
+			mounter.IsMountPointPartitionPath = "/dev/sdf"
+			mounter.IsMountPointResult = true
+			mounter.IsMountPointErr = nil
+			cmdRunner.AddCmdResult(
+				"blockdev --getra /dev/sdf",
+				fakesys.FakeCmdResult{Stdout: "256"},
+			)
+			options.PersistentDiskMigrationReadahead = 1024
+		})
+
+		It("migrates persistent disk with readahead override", func() {
+			err := platform.MigratePersistentDisk("/from/path", "/to/path")
```

Comment:
you can use `SetPersistentDiskMigrationReadahead` for this test to see that p.options isnt actually persistented after modification.